engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/ProcessDefinitionExtractor.scala

```diff
@@ -81,10 +81,16 @@ object ProcessDefinitionExtractor {
     import net.ceedubs.ficus.readers.ArbitraryTypeReader._
     import net.ceedubs.ficus.readers.ValueReader
 
-    implicit val nodeConfig: ValueReader[ParameterEditor] = ValueReader.relative(config => {
+    implicit val paramEditorReader: ValueReader[ParameterEditor] = ValueReader.relative(config => {
       val json = config.root().render(ConfigRenderOptions.concise().setJson(true))
-      CirceUtil.decodeJsonUnsafe[ParameterEditor](json, "invalid process config")
+      CirceUtil.decodeJsonUnsafe[ParameterEditor](json, "invalid parameter editor config")
     })
+
+    implicit val paramValidatorReader: ValueReader[ParameterValidator] = ValueReader.relative(config => {
+      val json = config.root().render(ConfigRenderOptions.concise().setJson(true))
```

Comment:
let's extract it some util? It's a bit hacky IMHO btw ;)

Reply:
Done